sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -240,16 +238,121 @@ public async Task EventHub_PartitionKey()
             var (jobHost, host) = BuildHost<EventHubPartitionKeyTestJobs>();
             using (jobHost)
             {
-                var method = typeof(EventHubPartitionKeyTestJobs).GetMethod("SendEvents_TestHub", BindingFlags.Static | BindingFlags.Public);
                 _eventWait = new ManualResetEvent(initialState: false);
-                await jobHost.CallAsync(method, new { input = _testId });
+                await jobHost.CallAsync(nameof(EventHubPartitionKeyTestJobs.SendEvents_TestHub), new { input = _testId });
 
                 bool result = _eventWait.WaitOne(Timeout);
 
                 Assert.True(result);
             }
         }
 
+        [Test]
+        public async Task EventHub_InitialOffsetFromStart()
+        {
+            var producer = new EventHubProducerClient(EventHubsTestEnvironment.Instance.EventHubsConnectionString, _eventHubScope.EventHubName);
+            await producer.SendAsync(new EventData[] { new EventData(new BinaryData(_testId)) });
+
+            var (jobHost, host) = BuildHost<EventHubTestInitialOffsetFromStartEndJobs>(
+                builder =>
+                    builder.ConfigureServices(services =>
+                    {
+                        services.Configure<EventHubOptions>(options =>
+                        {
+                            options.InitialOffsetOptions.Type = "FromStart";
+                            options.AddSender(_eventHubScope.EventHubName, EventHubsTestEnvironment.Instance.EventHubsConnectionString);
```

Comment:
You can call the default method manually: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs#L392